src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationIntTest.kt

```diff
@@ -81,98 +105,50 @@ class ActivitiesMigrationIntTest : SqsIntegrationTestBase() {
     }
 
     @Test
-    fun `will start processing pages of activities`() {
-      nomisApi.stubGetInitialCount(ACTIVITIES_ID_URL, 7) { activitiesIdsPagedResponse(it) }
-      nomisApi.stubMultipleGetActivitiesIdCounts(totalElements = 7, pageSize = 3)
-      nomisApi.stubMultipleGetActivities(1..7)
-      mappingApi.stubAllMappingsNotFound(ACTIVITIES_GET_MAPPING_URL)
-      mappingApi.stubMappingCreate(ACTIVITIES_CREATE_MAPPING_URL)
-      activitiesApi.stubCreateActivityForMigration()
-      activitiesApi.stubGetActivityCategories()
-      mappingApi.stubActivitiesMappingByMigrationId(count = 7)
-
-      webTestClient.post().uri("/migrate/activities")
-        .headers(setAuthorisation(roles = listOf("ROLE_MIGRATE_ACTIVITIES")))
-        .header("Content-Type", "application/json")
-        .body(BodyInserters.fromValue("""{ "prisonId": "BXI" }"""))
-        .exchange()
-        .expectStatus().isAccepted
+    fun `will migrate several pages of activities`() {
+      stubMigrationDependencies(7)
```

Comment:
The only changes to the existing actiivty migration tests is some refactoring and a tidy.